crates/runtime/src/databackend.rs

```diff
@@ -69,6 +72,12 @@ impl DataBackendBuilder {
         self
     }
 
+    #[must_use]
+    pub fn secrets(mut self, secrets: Option<Secret>) -> Self {
```

Comment:
let's name it `secret` to avoid confusion

Reply:
Name updated to secret in the recent updates